File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        map<int,int> m;
        for(int num : nums){
            m[num]++;
        }
        vector<pair<int,int>> a;
        for(const auto &p : m){
            a.push_back({p.second,p.first});

        }
        sort(a.rbegin(),a.rend());
        vector<int> res;
        for(int i=0;i<k;i++){
            res.push_back(a[i].second);
        }
        return res;
        
        
    }
};

File: Data Structures & Algorithms/subsets-ii/submission-0.py
class Solution:
    def subsetsWithDup(self, nums: List[int]) -> List[List[int]]:
        res=set()
        subset=[]
        nums.sort()

        def dfs(i,subset):
            if i>=len(nums):
                res.add(tuple(subset))
                return
            subset.append(nums[i])
            dfs(i+1,subset)
            subset.pop()
            dfs(i+1,subset)
        dfs(0,[])
        return [list(s) for s in res]